doc/reference/tt_cli/arguments.rst

```diff
@@ -0,0 +1,35 @@
+tt CLI arguments
```

Comment:
```suggestion
Arguments
```
Same as with the installation page. The user sees in the tree on the left that it's an instruction for tt CLI, so duplicating the name is not necessary.